packages/expo-constants/src/Constants.ts

```diff
@@ -132,21 +124,6 @@ Object.defineProperties(constants, {
     },
     enumerable: false,
   },
-  manifest: {
```

Comment:
Deprecation here was a misnomer on my part, sorry about that. We deprecated OTA/classic update manifest access via this field, but not embedded manifest access. So the current state of the field is correct (we can remove @deprecated from the docblock) and we want to leave this code, but can remove the `warnedAboutManifestField` warning thing.

All that being said, it's still preferable to use `Constants.expoConfig`, `Constants.expoGoConfig`, etc, so that note in the docblock should remain.